src/utils.ts

```diff
@@ -5,14 +5,28 @@ function objectFromEntries(keys: any, values: any): any {
   );
 }
 
-function getLastActiveRowIndex(column: string): number {
-  const runSheet: GoogleAppsScript.Spreadsheet.Sheet =
-    SpreadsheetApp.getActiveSpreadsheet().getSheetByName("Run");
-  const index: number = runSheet
-    .getRange(column + (runSheet.getLastRow() + 1))
-    .getNextDataCell(SpreadsheetApp.Direction.UP)
-    .getRow();
-  return index;
+function getLastActiveRowNumber(column: string, sheet): number {
+  let columunValues: string[] = transpose(
+    sheet.getRange(`${column}1:${column}`).getValues(),
+  )[0];
```

Comment:
do you really need to transpose? I guess that no. Also can use `Array.find` once you have the full data array